lib/ecto/changeset.ex

```diff
@@ -18,14 +19,14 @@ defmodule Ecto.Changeset do
 
   Changesets allow working with two kinds of data:
 
-    * internal to the application - for example programmatically generated,
-      or coming from other subsystems. This use case is primarily covered
-      by the `change/2` and `put_change/3` functions.
-
-    * external to the application - for example data provided by the user in
+    * external to the application - for example user input from
```

Comment:
The external case seems to be the one that a new user will encounter chronologically first. And also everywhere except the bullets, external is mentioned prior to internal, so I reordered them to be consistent.